### DISCUSSION TITLE
Futbol turnuvası sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#263] [Zeka Sorusu] [answered]
10 takımlı bir futbol liginde her takım diğer takımlarla ikişer maç yapmıştır. Ligin sonunda puan cetveli incelendiğinde şu sonuç farkedilmiştir: Tüm takımların galibiyet sayıları toplamı, tüm takımların beraberlik sayıları toplamına eşittir.
Buna göre ligde kaç maç berabere bitmiştir.

**Comment by Mertcakm** [comment]
30

**Reply by sonsuzus** [reply]
Soruları açıklamalı çözelim.

**Reply by Mertcakm** [reply, marked as answer]
Toplamda 90 maç yapılır galibiyet sayısı beraberlik sayısına eşit olduğuna ve galibiyet sayısı kadar mağlubiyet sayısıda olduğuna göre 

Mağlubiyet sayısı = Galibiyet sayısı = Beraberlik sayısı

Diye biliriz ve hepsi eşit olduğu için de 

x = x = x diye yazabiliriz

ve toplam x beraberlik sayısına ulaşılabilmesi için x/2 berabere kalınan maç olmalıdır. x mağlubiyet ve galibiyet olması içinse x berabere bitmeyen maç gerekir. Berabere biten maçla bitmeyen maçı toplarsakta toplam maç sayısına ulaşırız yani x + x/2 = 90 ve bu denklemden x 60 çıkar fakat berabere kalınan maç sayısı sorulduğu için 60/2 işleminden sonuç 30 çıkar.
